### DISCUSSION TITLE
Touch-sliding action for featured post on home page

### DISCUSSION THREAD

**Discussion by evandde** [hbstack/.github#20] [Ideas]
In mobile, an user must touch the small circle to see other featured post. 

![image](https://github.com/hbstack/.github/assets/57508537/29cfe9f5-8dbf-4d2e-a710-5495d72bb50d)

There is an arrow button in PC version, but not in mobile version.

I suggest to use 'touch-sliding action' to solve this problem.

Will it possible?

**Comment by razonyang** [comment]
It seems ok to me, since it pass the accessibility test of Lighthouse.

> Will it possible?

I think yes, but not plan on it so far.

**Comment by razonyang** [comment]
Available on `github.com/hbstack/carousel` `v0.2.2`.

```sh
$ hugo mod get -u github.com/hbstack/carousel
go: downloading github.com/hbstack/carousel v0.2.2
go: upgraded github.com/hbstack/carousel v0.2.1 => v0.2.2
```

**Reply by evandde** [reply]
Works well! Thanks a lot
